db/worker_jobs_migrate/20210809201322_create_good_jobs_separate_database.rb

```diff
@@ -0,0 +1,28 @@
+# same as 20210804180216_create_good_jobs.rb but runs in the worker database
+class CreateGoodJobsSeparateDatabase < ActiveRecord::Migration[5.2]
```

Comment:
note in this one it's in `db/worker_jobs_migrate` instead of `db/migrate` or `db/primary_migrate`